include/Config_WiFi.h

```diff
@@ -19,8 +19,16 @@ COPYRIGHT (c) 2017-2019 Mike Dunston
 //
 // DEFINE WiFi Parameters
 //
-#define SSID_NAME "SSID_NAME_HERE"
-#define SSID_PASSWORD "SSID_PASSWORD_HERE"
+#ifdef CONFIG_SSID_NAME
```

Comment:
should we consider moving these defines to ESP32CommandStation.h just after the sdkconfig.h include (both for this and MotorBoard) and then in Config.h when CMAKE is used we don't include WiFi/MotorBoard?

Reply:
your call. There are more settings which could be handled by the 'menuconfig' system, so your suggestion would probably reduce the typing. On the other hand I am following the pattern used in the espressif examples, so people might be used to it.

Reply:
I'm fine with moving these (and others) to the sdkconfig.h automatic-handling (maybe just exclude Config.h entirely when CMAKE is used!)

The only concern I have with CMAKE is that it doesn't seem that it is directly supported in PIO with VS Code but it is supported in CLion so there is some hope that it would work in VS Code as well.